job.py

```diff
@@ -0,0 +1,85 @@
+import shutil
+from urllib.request import urlopen
+from pathlib import Path
+import ssl
+import json
+from logger import logger
+
+
+def coroutine(f):
+    def wrap(*args, **kwargs):
+        gen = f(*args, **kwargs)
+        gen.send(None)
+        return gen
+    return wrap
+
+
+def get_and_write_data(condition, url):
+    context = ssl._create_unverified_context()
+    file = 'punchline.txt'
+    with condition:
+        with urlopen(url, context=context) as req:
+            resp = req.read().decode("utf-8")
+            resp = json.loads(resp)
+        if req.status != 200:
+            raise Exception(
+                "Error during execute request. {}: {}".format(
+                    resp.status, resp.reason
+                )
+            )
+        data = resp
+        if isinstance(data, dict):
+            path = Path(file)
+            setup = data['setup']
+            punchline = data['punchline']
+            print(
+                f'Setup: {setup} \n'
+                f'Punchline: {punchline}'
+            )
+            with open(path, mode='a') as config:
+                config.write(str(data))
+        else:
+            logger.error(type(data))
```

Comment:
вот это плохо, лучше тогда рейзить так 
```suggestion
           error_message = f"Wrong data type: {type(data)}! Dict expected."
           logger.error(error_message)
           raise ValueError(error_message)
```